src/binder.rs

```diff
@@ -78,6 +78,7 @@ pub struct BoundSubqueryTableReferenceAST {
 #[derive(Debug, PartialEq, Eq, Clone)]
 pub struct BoundInsertStatementAST {
     pub table_name: String,
+    pub column_names: Vec<String>,
```

Comment:
@mnmandahalf 
Thank you for your contribution.
I suggest using the Option type with None for cases where column_names is not specified, rather than implicitly indicating it with an empty Vec.